### DISCUSSION TITLE
Sonoff Pow R2 Modul & KNX

### DISCUSSION THREAD

**Discussion by wb86** [arendst/Tasmota#11395] [Q&A] [answered]
Hi,
I am using a Sonoff Pow R2 Modul with the KNX tool.
Everything works fine... exept the Energy Yesterday notification.

On the Tasmota Main Menu:
Energy Yesterday 0.002 kWh
Energy Total 0.013 kWh

Also the Console:
`15:03:10.463 KNX: Energy Total sent to 0.5.8 
15:03:10.466 KNX: Energy Yesterday sent to 0.5.7 
15:03:10.470 RSL: SENSOR = {"Time":"2021-03-20T15:03:10","ENERGY":{"TotalStartTime":"1970-01-01T00:00:04","Total":0.013,"Yesterday":0.002,"Today":0.007,"Period":0,"Power":5,"ApparentPower":11,"ReactivePower":10,"Factor":0.44,"Voltage":232,"Current":0.048}}`

But my ETS:
`177 | 20.03.2021 15:03:09,877 | vom Bus | S=7 | Niedrig | 1.2.0 |   | 0/5/8 |   | 6 | GroupValueWrite |   | 3C 56 A1 62 \| 0,0131 |  
178 | 20.03.2021 15:03:09,982 | vom Bus | S=0 | Niedrig | 1.2.0 |   | 0/5/7 |   | 6 | GroupValueWrite |   | 00 00 00 00 \| 0 |  `

Where is the mistake?

Tim

**Comment by ascillato** [comment, marked as answer]
Hi, in order to help you troubleshoot your issue, more information is needed. Please, open a new issue and complete the troubleshooting template. Please, don't forget to share the logs of the console. Please use weblog set to 4. Also, please share the screenshots of your KNX config in Tasmota. Thanks.
